grade_a1.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
```

Comment:
Why?

Reply:
I used it to test assignment 1 before I did assignment 2, and then I left it there. Should I need to delete it?